repair/featurize/__init__.py

```diff
@@ -3,11 +3,10 @@
 from .initfeat import InitFeaturizer
 from .initsimfeat import InitSimFeaturizer
 from .freqfeat import FreqFeaturizer
-from .occurfeat import OccurFeaturizer
 from .constraintfeat import ConstraintFeat
 from .langmodel import LangModelFeat
 from .initattfeat import InitAttFeaturizer
 from .occurattrfeat import OccurAttrFeaturizer
 
 __all__ = ['FeaturizedDataset', 'Featurizer', 'InitFeaturizer', 'InitSimFeaturizer', 'FreqFeaturizer',
-           'OccurFeaturizer', 'ConstraintFeat', 'LangModelFeat', 'InitAttFeaturizer', 'OccurAttrFeaturizer']
+           'ConstraintFeat', 'LangModelFeat', 'InitAttFeaturizer', 'OccurAttrFeaturizer']
```

Comment:
Can we rename `InitAttFeaturizer` to `InitAttrFeaturizer`?

Reply:
Done